packages/stryker/src/Stryker.ts

```diff
@@ -81,7 +81,8 @@ export default class Stryker {
       this.reporter.onScoreCalculated(score);
       ScoreResultCalculator.determineExitCode(score, this.config.thresholds);
       await this.wrapUpReporter();
-      await StrykerTempFolder.clean();
+      TempFolder.instance().initialize();
```

Comment:
I think this one should be moved up right? Right before the `initialTestRun`?

Reply:
Fixed